setup.py

```diff
@@ -3,6 +3,13 @@
 
 setup(name='youclid',
       version='0.1',
+      python_requires='>=3.6',
       packages=find_packages(),
-      install_requires=[]
+      install_requires=[],
+      package_data={'youclidbackend': ['data/template.html',
+                                       'data/default.css',
+                                       'data/draw.js',
+                                       'data/index.js']},
+      include_package_data=True,
```

Comment:
The `package_data` will ship our relevant files with the install. The key is the package where the data files live. Note that this means these files MUST exist inside our pacakge (hence the symlinks to the files in the frontend).